### DISCUSSION TITLE
PCB Source

### DISCUSSION THREAD

**Discussion by MrAges** [nliaudat/esp32_8ch_motor_shield#49] [General]
Hey,

is there maybe a chance to get the PCB Source back again. I´m very interrested in this great work and i want to get some of these boards manufactured. 

thanks.

**Comment by nliaudat** [comment]
If you want to manufacture boards, all you need is in https://github.com/nliaudat/esp32_8ch_motor_shield/releases

If you want to contribute, you must have skills in electronics and if I can share you the project.

**Reply by MrAges** [reply]
ok i got it. thanks.
